include/morphio/enums.h

```diff
@@ -151,7 +151,8 @@ enum VascularSectionType
     SECTION_ARTERIOLE = 4,
     SECTION_VENOUS_CAPILLARY = 5,
     SECTION_ARTERIAL_CAPILLARY = 6,
-    SECTION_TRANSITIONAL = 7
+    SECTION_TRANSITIONAL = 7,
+    SECTION_CUST_START = 8
```

Comment:
CUSTOM

Reply:
that clashes with the enum in SECTION_TYPE

Reply:
CUSTARD?

Reply:
https://stackoverflow.com/a/23288968/2533394